bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/BooleanItemReader.java

```diff
@@ -54,23 +55,30 @@
     }
 
     Boolean getValue() {
```

Comment:
Why not make this primitive?
```suggestion
    boolean getValue() {
```